os_migrate/roles/import_workloads/defaults/main.yml

```diff
@@ -4,5 +4,6 @@ os_migrate_conversion_keypair_private_path: "{{ os_migrate_data_dir }}/conversio
 os_migrate_timeout: 1800
 os_migrate_workload_cleanup_on_failure: true
 os_migrate_workload_boot_volume_prefix: os-migrate-
+os_migrate_workload_stop_before_migration: false
```

Comment:
+1 for defaulting this to `false` in production. However, we should default to `true` in tests, here:

https://github.com/os-migrate/os-migrate/blob/92c359e4e726ec53f2c0a519c781917bbe60cb68/tests/e2e/tenant/scenario_variables.yml

And we should remove this:

https://github.com/os-migrate/os-migrate/blob/92c359e4e726ec53f2c0a519c781917bbe60cb68/tests/e2e/tenant/seed_workload.yml#L33-L43

That way your new code here will be tested by the e2e tests/CI.

Reply:
Thank you. Changes made in https://github.com/os-migrate/os-migrate/pull/464/commits/96bed12fd77332717c5c0e5c4c35f13e5f1a900f and https://github.com/os-migrate/os-migrate/pull/464/commits/8434ba1fd306dafbbcd9e912762afee7cc4f143c